src/main/java/seedu/stock/commons/util/SortUtil.java

```diff
@@ -0,0 +1,130 @@
+package seedu.stock.commons.util;
+
+import java.math.BigInteger;
+import java.util.Comparator;
+
+import seedu.stock.model.stock.Stock;
+
+public class SortUtil {
+
+    public enum Order {
+        ASCENDING, DESCENDING
+    }
+
+    public enum Field {
+        NAME, SOURCE, LOCATION, QUANTITY, SERIALNUMBER
+    }
+
+    /**
+     * Returns a description of the sorted field.
+     *
+     * @param field The sorted field.
+     * @return The description of the sorted field.
+     */
+    public static String getFieldDescription(Field field) {
+        switch (field) {
+        case NAME:
+        case QUANTITY:
+        case SOURCE:
+        case LOCATION:
+            return field.toString().toLowerCase();
+        case SERIALNUMBER:
+            return "serial number";
+        default:
+            return "";
```

Comment:
Will this case ever be reached? If not, perhaps adding an assertion can be more defensive!